### PR TITLE
Add espeak + a few other tools from #docker wishlist

### PR DESCRIPTION
**Pull request by gavincangan**
- [ ] Tested on robot

### CODE REVIEW

File: drake-torch.dockerfile [hunk 1]
```diff
@@ -279,6 +282,13 @@ RUN apt-get update && apt-get install -y \
 RUN pip3 install --ignore-installed pyzmq
 RUN pip3 install jupyter
 
+# install nice-to-have some dev tools
+RUN apt-get -y update && apt-get -y upgrade && apt-get install -q -y \
+    clang-format \
```
Comment (sprax) [inline drake-torch.dockerfile]: will this get the latest version?
clang-format 8 or 9?

**Comment by 5yler** [comment]
If possible I would prefer to expand this to include a few more additions:
```
ros-melodic-tf-conversions
```
as well as some additional prereqs for `ElasticFusion`

File: drake-torch.dockerfile (hunk 1 above)
Comment (gavincangan) [reply drake-torch.dockerfile]: Looks like this installs `clang-format-6`

Comment (gavincangan) [reply drake-torch.dockerfile]: I'll replace this with `clang-format-8`, then?

Comment (5yler) [reply drake-torch.dockerfile]: Please update and move all to bottom.

**Pull request merged**